src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  box-sizing: border-box;
+
+  &__photo {
+    background-image: url(..//images/imac.jpeg);
```

Comment:
The URL for the background image contains a double slash ('..//'). It should be corrected to '../images/imac.jpeg'.